nntrainer/compiler/recurrent_realizer.cpp

```diff
@@ -184,8 +184,7 @@ static void propagateTimestep(LayerNode *node, unsigned int time_step,
 
   /** @todo add an interface to check if a layer supports a property */
   auto is_recurrent_type = [](LayerNode *node) {
-    return node->getType() == RNNCellLayer::type ||
-           node->getType() == ZoneoutLSTMCellLayer::type;
+    return node->getType() == ZoneoutLSTMCellLayer::type;
```

Comment:
how about move this to layer_node.h?

Reply:
Actually the propagateTimestep function will be removed in following pr